lib/api/BackbeatAPI.js

```diff
@@ -113,6 +135,114 @@ class BackbeatAPI {
         });
     }
 
+    /**
+     * Get replication backlog in bytes and count
+     * @param {function} cb -callback(error, data)
+     * @return {undefined}
+     */
+    backlog(cb) {
+        async.series([
+            next => this._queryStats(OPS, next),
+            next => this._queryStats(BYTES, next),
+        ], (err, res) => {
+            if (err) {
+                this._logger.trace('error occurred getting backlog');
+                return cb(errors.InternalError);
+            }
+            const data = {
+                backlog: {
+                    details: 'Replication backlog in bytes and count',
+                    count: res[0].requests,
+                    bytes: res[1].requests,
+                },
+            };
+            return cb(null, data);
+        });
+    }
+
+    /**
+     * Get completed replicated stats by bytes and count
+     * @param {function} cb - callback(error, data)
+     * @return {undefined}
+     */
+    completions(cb) {
+        async.series([
+            next => this._queryStats(OPS_DONE, next),
+            next => this._queryStats(BYTES_DONE, next),
+        ], (err, res) => {
+            if (err) {
+                this._logger.trace('error getting replication completions');
+                return cb(errors.InternalError);
+            }
+            const data = {
+                completions: {
+                    details: 'Replciations completed in bytes and count in ' +
+                        `the last ${EXPIRY} seconds`,
+                    count: res[0].requests,
+                    bytes: res[1].requests,
+                },
+            };
+            return cb(null, data);
+        });
+    }
+
+    /**
+     * Get current throughput (ops/sec)
+     * @param {function} cb - callback(error, data)
+     * @return {undefined}
+     */
+    throughput(cb) {
+        this._queryStats(OPS_DONE, (err, res) => {
+            if (err) {
+                this._logger.trace('error getting throughput');
+                return cb(errors.InternalError);
+            }
+            const throughput = (res.requests / res.sampleDuration).toFixed(2);
+            const data = {
+                throughput: {
+                    details: 'CRR throughput in ops/sec',
+                    value: throughput,
+                },
+            };
+            return cb(null, data);
+        });
+    }
+
+    /**
+     * Get current bandwidth (MBs/sec)
+     * @param {function} cb = callback(error, data)
+     * @return {undefined}
+     */
+    bandwidth(cb) {
+        this._queryStats(BYTES_DONE, (err, res) => {
+            if (err) {
+                this._logger.trace('error getting bandwidth');
+                return cb(errors.InternalError);
+            }
+            const bandwidth = Math.floor(
```

Comment:
Why double `Math.floor`?

Actually you could provide some decimal digits since MB/s is kind of rough precision (especially if you CRR to a distant public cloud), though 2 or 3 more decimals should be enough. So you could just get rid of Math.floor and compute with a simple division, or use `toFixed()` if you wish but outputting a plain full precision floating point value should be fine.